design-docs/CLI-Subscriptions.md

```diff
@@ -175,6 +175,49 @@ In the case of invalid options, print an error message (stderr) and set the exit
 code to 2. In the case of an API error response, print the response JSON and 
 set code to 1.
 
+## Request
+
+### Interface
+
+```
+planet subscriptions request [OPTIONS]
+
+Generate a subscriptions request.
+
+This command provides support for building the subscription request JSON used to create or 
+update a subscription. It outputs the subscription request.
+
+Options:
+  --name TEXT                     Subscription name. Does not need to be unique.
+                                  [required]
+  --asset-types TEXT              One or more comma-separated asset types [required]
+  --item-types TEXT               One or more comma-separated item-types [required]
+  --geometry JSON                 GeoJSON of area to deliver subscription to. Use 
+                                  --clip only deliver pixels that match this area.
+                                  Can be a json string, filename, or '-' for stdin.
+  --clip                          Use the geometry GeoJSON as the clip geometry for 
+                                  the subscription.
+  --start-time DATETIME           Start date and time to begin subscription.
+  --end-time DATETIME             Date and time to end the subscription.
+  --rrule TEXT                    iCalendar recurrance rule to specify recurrances. 
+  --filter JSON                   A search filter can be specified a json string, 
```

Comment:
This ideally would let people reuse the `planet data filter` command, but I'm not sure of the proper behavior if someone tries to use a filter with a geometry or a time range. Should figure out what the subscriptions API does in that case. 

There also might be some additional convenience we could enable by tapping into the geometry and date time filters, but that might get too complicated.

Reply:
From https://developers.planet.com/docs/subscriptions/source/#filter-details
> A subscription can leverage all filter types (with the exception of DateRangeFilter and GeometryFilter).

So the subscriptions API forbids use of filters containing geometry or time ranges